premise/ecoinvent_modification.py

```diff
@@ -49,9 +49,16 @@
     load_constants,
     print_version,
     warning_about_biogenic_co2,
-    write_brightway2_database,
 )
 
+try:
+    import bw_processing
+
+    from .brightway25 import write_brightway_database
+except ImportError:
```

Comment:
suggestion (non-blocking): It would be useful to warn the user if the import goes OK or (bw2) or wrong.

Reply:
Added to `todo`